content/plugins/host-pool.md

```diff
@@ -73,59 +70,58 @@ Base type for a pool host.
 
 **Properties:**
 
-  * `os` windows.
+  * `os` - The Windows operating system.
```

Comment:
This is an overridden default for the `os` property (because `...WindowsHost` derives from `...Host`), and it is literally "windows". Given that this is a property that shouldn't be overridden ever, perhaps we should make that clear instead, e.g.
```markdown
* `os`: `windows`  (this is an implementation detail and should not be overridden)
```